conf/susi/susi_cognition_000.json

```diff
@@ -712,6 +712,16 @@
 			"actions":[	{"type":"answer", "select":"random", "phrases":[
 							"I like to chat with people!"
 						]}]
-		}
+		},
+		{
+			"keys"   :["github"],
+			"score"  :3000,
+			"example": "Susi, View the Github profile of mariobehling",
+			"phrases":[ {"type":"pattern", "expression":"* github * of *"}],
```

Comment:
Any specific reason `* github * of *` ?
As far as I know the possible queries can be
- Show me the github profile of mariobehling
- What are the github profile stats of mariobehling
- Read mariobehling github profile 
- Give me more information from github about mariobehling
- Can you find mariobehling on github and give me information